hack/build-image/Dockerfile

```diff
@@ -12,40 +12,38 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.12
+FROM golang:1.13
 
+# Use a proxy for go modules to reduce the likelihood of various hosts being down and breaking the build
+ENV GOPROXY=https://proxy.golang.org
+
+# get code-generation tools (for now keep in GOPATH since they're not fully modules-compatible yet)
 RUN mkdir -p /go/src/k8s.io
 WORKDIR /go/src/k8s.io
 RUN git config --global advice.detachedHead false
 RUN git clone -b kubernetes-1.17.0 https://github.com/kubernetes/code-generator
-RUN git clone -b kubernetes-1.17.0 https://github.com/kubernetes/apimachinery 
-# Use a proxy for go modules to reduce the likelihood of various hosts being down and breaking the build
-ENV GOPROXY=https://proxy.golang.org
 # vendor code-generator go modules to be compatible with pre-1.15
-WORKDIR  /go/src/k8s.io/code-generator
+WORKDIR /go/src/k8s.io/code-generator
 # Don't use ENV here because we don't want to disable modules for subsequent commands
 RUN GO111MODULE=on go mod vendor
```

Comment:
Should we change this to use the env var now that we are cutting over to go modules?